### ISSUE TITLE
lime-docs stop using SVN

### ISSUE THREAD

**Issue by ilario** [libremesh/lime-packages#1054]
In order to compile lime-docs, the host has to be online and SVN has to be installed.
This is annoying and an option using git would be better.
Likely, this would require the download to the whole libremesh website in order to copy only the docs folder.
See discussion on #1053

**Comment by ilario** [comment]
@LaneaLucy reported that Github stopped offering SVN:

```
svn: E170013: Unable to connect to a repository at URL 'https://github.com/libremesh/lime-web/trunk/docs'
svn: E900002: Subversion support temporarily disabled.

GitHub has temporarily disabled Subversion support as part of a planned brownout
in order to give you advance notice of the upcoming permanent removal of
Subversion support on 2024-01-08.

Please see https://github.blog/2023-01-20-sunsetting-subversion-support/ for details.
```

So we need to have #1053 proposed again @a-gave

**Comment by ilario** [comment]
Should have been fixed merging #1053. Please report if the issue is still happening.
